doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-kogito-cli.adoc

```diff
@@ -0,0 +1,88 @@
++
+.SmallRye Heath dependency for Quarkus applications on OpenShift
+[source,xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-smallrye-health</artifactId>
+</dependency>
+----
+
+.Procedure
+. Go to the https://github.com/kiegroup/kogito-operator/releases[`kogito-operator`] releases page in GitHub and download the latest version of the `kogito-cli-_RELEASE_` binary file that is specific to your operating system.
+. Extract the `kogito-cli-_RELEASE_` binary file to a local directory:
++
+--
+* On Linux or Mac: In a command terminal, navigate to the directory where you downloaded the `kogito-cli-_RELEASE_` binary file and enter the following command to extract the contents:
++
+.Extract the {KOGITO} CLI distribution
+[source,subs="attributes+,+quotes"]
+----
+$ tar -xvf kogito-cli-_RELEASE_.tar.gz
+----
+
+* On Windows: In your file browser, navigate to the directory where you downloaded the `kogito-cli-_RELEASE_` binary file and extract the ZIP file.
+
+The `kogito` executable file appears.
+--
+. Move the extracted `kogito` file to an existing directory in your `PATH` variable:
++
+--
+* On Linux or Mac: In a command terminal, enter the following command:
++
+.Move the `kogito` file
+[source,subs="attributes+,+quotes"]
+----
+$ cp /__PATH_TO_KOGITO__ /usr/local/bin
+----
+
+* On Windows: Update the relevant `PATH` environment variables in your system settings to include the path to the {KOGITO} CLI folder. For example, on Windows 10, go to *Settings* -> *System Info* -> *Advanced System Settings* -> *Advanced* -> *Environment Variables* and in the *User* or *System* variables, add the path for the {KOGITO} CLI folder to the `PATH` variable. Close and reopen your Windows command prompt to apply the changes.
+--
+. With the {KOGITO} CLI now installed, enter the following commands to deploy your {KOGITO} microservices on OpenShift from source:
++
+--
+NOTE: You must be logged in to the relevant OpenShift cluster using the `oc login` command.
+
+.Example {KOGITO} microservice deployment from existing namespace
+[source,subs="attributes+,+quotes"]
+----
+// Uses the provisioned namespace in your OpenShift cluster
+$ kogito use-project __PROJECT_NAME__
+
+// Deploys a new {PRODUCT} service from a Git source
+$ kogito deploy-service example-quarkus https://github.com/kiegroup/kogito-examples --context-dir process-quarkus-example
+----
+
+The {KOGITO} Operator uses the default branch in the specified Git repository, usually `master`.
+
+NOTE: The first time that you use the {KOGITO} CLI to interact with a project or service, the {KOGITO} Operator is automatically installed and used to execute the relevant tasks.
```

Comment:
```suggestion
NOTE: The first time you use the {KOGITO} CLI to interact with a project or service, the {KOGITO} Operator is automatically installed and used to execute the relevant tasks.
```

Reply:
Let's leave it up to Heena. I prefer keeping 'that'.